### ISSUE TITLE
Calendar Not Syncing

### ISSUE THREAD

**Issue by ababs4077** [HannahMitt/HomeMirror#44]
Next calendar event is not appearing.. anyone else having this problem?

**Comment by HannahMitt** [comment]
Is the calendar event for the same day and visible in Google Calendar?

**Comment by ababs4077** [comment]
Yes... same day and visible. I've tried with all day events, specific timed events, and made sure the calendar is syncing correctly. I'm away from my main computer now but will update when I'm back.

**Comment by ababs4077** [comment]
Complete reinstall fixed the issue for me. Hadn't made any changes to your code previously. Chalking it up to something on my end, perhaps with Calendar settings!

**Comment by mdzierza** [comment]
Hi Hannah, 
What a great project. In this topic however, I also can't seem to sync the next calendar event. The events are in my google calendar and I did try reinstall as well.
On another topic, News feed only feeds first headline not all of them consecutively. Is this by design?

**Comment by HannahMitt** [comment]
Hi @mdzierza . Sorry its hard to guess what the calendar issue might be, but I'll keep an eye out.

The news module is very simple and does only show the first headline. If you'd like to suggest another way it should work feel free to open another issue for that!

**Comment by chippxero** [comment]
Same issue, at present I am putting it down to the tablet/android version I am using (samsung GT-P7510 on 4.0.4)
